nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSStateFlattener.java

```diff
@@ -36,18 +37,37 @@
  */
 public class DBSStateFlattener {
 
-    public static Map<String, Serializable> flatten(State state) {
+    private final Map<String, String> keyMappings;
```

Comment:
Please use `protected` and not `private` for fields. You never know when subclassing something can be useful.

 (except `log` and `serialVersionUID` which should never be shared even by subclasses)